src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -87,11 +86,13 @@ public void run(RunConfiguration config) throws AppEngineException {
               .getAbsolutePath();
       jvmArguments.add("-javaagent:" + appengineAgentJar);
     } 
-    arguments.addAll(DevAppServerArgs.get("server", config.getHost()));
-    arguments.addAll(DevAppServerArgs.get("address", config.getPort()));
-    arguments.addAll(DevAppServerArgs.get("jvm_flag", config.getJvmFlags()));
```

Comment:
I'm still not convinced this is true, running the script from the command line 
```
 ./dev_appserver.sh --jvm_flag=-Xdebug --
jvm_flag=-Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005 --allow_remote_shutdown /path/to/some/app
```
will work and start the server in debug.

I will try from the library and see though.